launcher/verifier/fake/fakeverifier.go

```diff
@@ -0,0 +1,60 @@
+// Package fake is a fake implementation of the Client interface for testing.
+package fake
+
+import (
+	"context"
+	"crypto"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/google/go-tpm-tools/launcher/verifier"
+)
+
+type fakeClient struct {
+	//pbClient servpb.AttestationVerifierClient
+	signer crypto.Signer
+}
+
+// NewClient contructs a new fake client given a crypto.Signer.
+func NewClient(signer crypto.Signer) verifier.Client {
+	return &fakeClient{signer}
+}
+
+// CreateChallenge returns a hard coded, basic challenge.
+//
+// If you have found this method is insufficient for your tests, this class must be updated to
+// allow for better testing.
+func (fc *fakeClient) CreateChallenge(ctx context.Context) (*verifier.Challenge, error) {
+	return &verifier.Challenge{
+		Name:  "FakeName",
+		Nonce: []byte{0x0},
```

Comment:
Can we have this Name and nonce at least look something like real data?

For example:
  - `Name:` `"projects/fakeProject/locations/fakeRegion/challenges/d882c62f-452f-4709-9335-0cccaf64eee1"`
  - `Nonce:` 16 bytes of random data

Reply:
Sure, let me know what you think of this